images/mta-operator.yml

```diff
@@ -0,0 +1,32 @@
+cachito:
+  packages:
```

Comment:
```suggestion
```
ref: https://github.com/openshift-eng/ocp-build-data/blob/mtc-1.8/images/openshift-migration-operator.yml

If we are not resolving any deps, no need for this.